<launch>
  <arg name="veh" doc="Name of vehicle. ex: megaman"/>
  <arg name="local" default="true" doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>

  <node pkg="line_sanity" type="line_sanity_node.py" name="line_sanity_node" output="screen" clear_params="true" required="true">
    <param name="vehicle" value="$(arg veh)" />
  </node>

  <!-- run lsd line detection -->
  <!-- <include file="$(find line_detector)/launch/line_detector_node.launch">
    <arg name="config" value="$(arg config)"/>
    <arg name="veh" value="$(arg veh)"/>
    <arg name="local" value="$(arg local)"/>
    <arg name="param_file_name" value="$(arg param_file_name)"/>
  </include> -->

  <!-- run ground projection -->
  <!-- <include file="$(find ground_projection)/launch/ground_projection.launch">
    <arg name="config" value="$(arg config)"/>
    <arg name="veh" value="$(arg veh)"/>
    <arg name="local" value="$(arg local)"/>
  </include> -->

  <!-- <group ns="$(arg veh)">


    <node if="$(arg local)"  pkg="odometry" type="odometry.py" name="odometry_node" output="screen" clear_params="true" required="true">
      <remap from="~wheels_cmd" to="wheels_driver_node/wheels_cmd_executed"/>
    </node>

    <node if="$(arg local)"  pkg="show_map" type="show_map.py" name="show_map_node" output="screen" clear_params="true" required="true">
      <remap from="~lineseglist_in" to="ground_projection/lineseglist_out_lsd"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find show_map)/rviz_conf/map_view.rviz"/>


  </group> -->
</launch>

